src/main/java/io/castle/client/model/RiskPolicyResult.java

```diff
@@ -0,0 +1,50 @@
+package io.castle.client.model;
+
+/**
+ * Information about risk policy used for the action.
+ * <p>
+ * See the documentation for the semantics of each case.
+ * It can be null.
+ *
+ * @see <a href="https://castle.io/docs/authentication">Adaptive authentication</a>
+ */
+public class RiskPolicyResult {
+
+    /**
+     * id of the risk policy
+     */
+    private String id;
+
+    /**
+     * id of the risk policy revision
+     */
+    private String revisionId;
+
+    /**
+     * name of the risk policy
+     */
+    private String name;
+
+    /**
+     * type of the risk policy
+     */
+    private String type;
```

Comment:
I think this should be an enum, not a string: https://github.com/castle/castle-risk_policies/blob/master/lib/castle/risk_policies/models/risk_policy/type.rb

When deserializing, we should make sure not to break client code if we introduce a new type, so I'd opt for logging a WARN and returning null for unknown values.